idaes/gas_solid_contactors/unit_models/bubbling_fluidized_bed.py

```diff
@@ -560,10 +560,16 @@ def _make_vars_params(self):
             solid_phase.property_package.get_metadata().get_derived_units
 
         # Declare Mutable Parameters
-        self.eps = Param(mutable=True,
-                         default=1e-8,
-                         doc='Smoothing Factor for Smooth IF Statements',
-                         units=pyunits.dimensionless)
+        self.eps_bulk = Param(mutable=True,
+                              default=1e-8,
```

Comment:
I strongly suggest you create two new module level constants to define these values at the beginning of the module:

```
EPD_BULK = 1e-8
EPS_CONV = 1e-8
```

Then, use these constants to initialize the `Params` when you create them (i.e. `default=EPS_BULK`).